### ISSUE TITLE
Clang support?

### ISSUE THREAD

**Issue by Unitrunker** [linkotec/ffts#10]
I submitted a PR for FreeBSD / clang support in CMakeLists.txt to anthonix in 2018. Is there any interest here? I can rebase and resubmit.

**Comment by Unitrunker** [comment]
Please consider using this fork's change instead:

https://github.com/linkotec/ffts/compare/master...mcourteaux:master

**Comment by emmenlau** [comment]
Yep, https://github.com/mcourteaux/master seems to work for clang also on Android with NDK r23b.
